tests/posix/threads_base/prj.conf

```diff
@@ -0,0 +1,15 @@
+CONFIG_POSIX_API=y
+CONFIG_ZTEST=y
+
+CONFIG_POSIX_AEP_CHOICE_BASE=y
+CONFIG_POSIX_THREADS=y
+CONFIG_POSIX_PRIORITY_SCHEDULING=y
+CONFIG_POSIX_THREAD_PRIORITY_SCHEDULING=y
```

Comment:
Both of these are Options rather than Option Groups, so we should remove them them here.

Additionally, `_POSIX_THREAD_PRIORITY_SCHEDULING` should be activated by the `XSI_REALTIME_THREADS` Option Group mentioned in https://github.com/zephyrproject-rtos/zephyr/issues/86061

The implication is that is someone were to call `pthread_attr_setschedparam()` or `pthread_attr_getschedparam()`, which are oddly still part of `POSIX_THREADS_BASE`, then those functions would return `ENOTSUP`. The `posix_threads_base` testsuite should also account for that.

https://pubs.opengroup.org/onlinepubs/9699919799/functions/pthread_attr_getschedparam.html
https://pubs.opengroup.org/onlinepubs/9699919799/xrat/V4_subprofiles.html